### ISSUE TITLE
[LionInputDatepicker] does not work with Safari + iOS Voiceover

### ISSUE THREAD

**Issue by ghost** [ing-bank/lion#1973]
## Reproduction

1. Enable Voiceover in iOS accessibility settings
2. Open the datepicker calendar
3. Try to click on any of the calendar buttons

## Expected behavior

With the iOS Voiceover screenreader, I can open the datepicker calendar and pick a date or modify the year/month.  

## Actual Behavior

Calendar itself is unresponsive, except for the close button on it.

## Additional context

Used an iPhone 7 Plus with iOS 15.7.5 and with the stock Voiceover screenreader.
Seems the bug is unrelated to https://github.com/ing-bank/lion/issues/1962
